tests/end2end/playwright/print.spec.js

```diff
@@ -615,3 +615,67 @@ test.describe('Print base layers', () => {
         expect(getPrintResponse?.headers()['content-type']).toBe('application/pdf');
     });
 });
+
+test.describe('Error while printing', () => {
+
+    test.beforeEach(async ({ page }) => {
+        const url = '/index.php/view/map/?repository=testsrepository&project=print';
+        await page.goto(url, { waitUntil: 'networkidle' });
+        /*
+        */
+    });
+
+    test('Print error', async ({ page }) => {
+        await page.locator('#button-print').click();
+
+        await page.locator('#print-scale').selectOption('100000');
+
+        await page.route('**/service*', async route => {
+            if (route.request()?.postData()?.includes('GetPrint'))
+                await route.fulfill({
+                    status: 404,
+                    contentType: 'text/plain',
+                    body: 'Not Found!'
+                });
+            else
+                await route.continue();
+        });
+
+        await page.locator('#print-launch').click();
+
+        await expect(page.getByText('Error while printing!')).toBeVisible();
```

Comment:
```suggestion
        await expect(page.getByText('Error while printing!')).toBeVisible();
```

use CSS selector instead of text ?